drivers/arm/gic/v3/gicv3_helpers.c

```diff
@@ -14,6 +14,7 @@
 
 #include "../common/gic_common_private.h"
 #include "gicv3_private.h"
+#include "arm_gicv3_common.h"
```

Comment:
This header contains implementation-defined values. Including it here would prevent other platforms from using this driver.

Reply:
Then how to handle this case? Add a function in our imx platform and if the function is avaiable, call it? Then other platforms will NOT be impacted?

Reply:
I'm thinking that maybe a build option is more in-line with what we do in the codebase (check ``PL011_GENERIC_UART``). If the definitions of that header are present in most arm designs, maybe this build option makes sense. I'm not a fan of either option to be honest.

Reply:
Looks like no better option, I added a build option for it.